java/src/org/openqa/selenium/grid/node/httpd/DefaultNodeConfig.java

```diff
@@ -28,7 +28,7 @@ class DefaultNodeConfig extends MapConfig {
             "node",
                 ImmutableMap.of(
                     // We use this instead of setting the default ports for
-                    // the publish and subscribe ports of the event bus so
```

Comment:
Why?

Reply:
@diemol, guess I got the spelling wrong
I mean:  `We use this instead of setting the default ports for the publishing and...`
`the publish (ports) and subscribe ports` were right

Reply:
i've updated the comment

Reply:
👍